Makefile

```diff
@@ -529,6 +550,10 @@ clean-book: ## Remove all generated GitBook files
 clean-bindata: ## Remove bindata generated folder
 	rm -rf $(GOBINDATA_CLUSTERCTL_DIR)/manifest
 
+.PHONY: clean-manifests ## Reset manifests in config directories back to master
+clean-manifests:
+	git checkout master config bootstrap/kubeadm/config controlplane/kubeadm/config test/infrastructure/docker/config
```

Comment:
Should this only be used during e2e running in CI?

Reply:
You need it locally if you want to build all the images and restore all the yaml's back to their original state. Not needed in CI because we're going to throw away the tree at the end.

Reply:
Ah, got it. Test machine node ref is nil, but not the leader candidate.

Reply:
Got it, what if a user locally isn't on master and had changes to those files?

Reply:
Ummmmmmm...I'll stick a pause and massive warning. Having something is better than manual clean up and accidentally filing a PR. I know @ncdc wants to change it so it doesn't even have this behaviour, and we use a temp dir in CAPV to avoid this issue.